commcare_connect/templates/opportunity/learn_progress.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+  <div class="container">
```

Comment:
These pages should include some context about which opportunity the user is looking at. For now perhaps just a title with the opportunity name.

In future I'm thinking this info could be displayed on an Opportunity Details page which would show general info about the opp and probably have  the progress info shown below.

It would also have buttons for editing etc. Here's an example from another product:

![image](https://github.com/dimagi/commcare-connect/assets/249606/cd4fec62-eca3-4420-a216-0cfce82584cd)

Reply:
I made some changes to clean up the Table and the UI to include the opportunity name. [92508b6](https://github.com/dimagi/commcare-connect/pull/60/commits/92508b60b355530905f76e56a9fcc6486c336be6)

Reply:
I was also thinking of renaming this page to the Opportunity Detail page and adding the learn progress table on this page.